nyaa/models.py

```diff
@@ -771,6 +771,44 @@ def __init__(self, info_hash, method):
         self.method = method
 
 
+class RangeBan(db.Model):
+    __tablename__ = 'rangebans'
+
+    id = db.Column(db.Integer, primary_key=True)
+    _cidr_string = db.Column(db.String(length=18), nullable=False)
```

Comment:
I haven't checked if SQA removes the leading `_` for the DB schema, but if it doesn't, consider `db.Column('cidr_string', db.String(length=18), nullable=False)` instead.